source/content/platform-considerations.md

```diff
@@ -73,6 +73,12 @@ Pantheon does not currently support LESS or Sass/Compass CSS preprocessor langua
 
 MySQL stored procedures are not supported. Due to the nature of the platform, there is no guarantee that they will  persist following a database migration. You can avoid the use of stored procedures by using parameterized queries or [object-relational mapping](https://en.wikipedia.org/wiki/Object-relational_mapping).
 
+## Drupal 7 and Ampersands
+
+A Drupal 7 site given a URL with an ampersand (&) in it will return a 404, regardless of the presence of a matching path.
```

Comment:
Let's be more precise. & will work if it's not used to separate query parameters, but not if it's part of the page path itself